packages/fxa-auth-server/fxa-oauth-server/lib/routes/token.js

```diff
@@ -368,7 +290,7 @@ async function validateRefreshTokenGrant(client, params) {
   // Does it belong to this client?
   if (!crypto.timingSafeEqual(tokObj.clientId, client.id)) {
     logger.debug('refresh_token.mismatch', {
-      client: params.client_id,
+      client: hex(client.id),
```

Comment:
This this just to work from the canonical source of truth rather than the params?

Reply:
Yes; previously we mutated `params` in-place when extracting the credentials, which felt a bit off after moving it into a separate function.